flaml/model.py

```diff
@@ -417,32 +417,18 @@ def _join(X_train, y_train, task):
     def search_space(cls, data_size, task, **params):
         search_space_dict = {
             "learning_rate": {
-                "domain": tune.choice(
-                    [1e-6, 2e-6, 4e-6, 8e-6, 16e-6, 32e-6, 64e-6, 128e-6]
-                ),
-                "init_value": 8e-6,
+                "domain": tune.loguniform(1e-6, 1e-4),
+                "init_value": 1e-5,
             },
             "num_train_epochs": {
-                "domain": tune.choice([1, 3, 5, 7, 9]),
+                "domain": tune.choice([1, 2, 3, 4, 5]),
                 "init_value": 3.0,  # to be consistent with roberta
             },
             "per_device_train_batch_size": {
-                "domain": tune.choice([4, 8, 16, 32]),
+                "domain": tune.choice([4, 8, 16, 32, 64]),
                 "init_value": 32,
             },
-            "warmup_ratio": {
-                "domain": tune.choice([0, 0.1, 0.2, 0.3]),
-                "init_value": 0.0,
-            },
-            "weight_decay": {
-                "domain": tune.choice([0, 0.1, 0.2, 0.3]),
-                "init_value": 0.0,
-            },
-            "adam_epsilon": {
-                "domain": tune.choice([1e-8, 1e-7, 1e-6]),
-                "init_value": 1e-6,
-            },
-            "seed": {"domain": tune.randint(40, 45), "init_value": 42},
+            "seed": {"domain": tune.randint(1, 40), "init_value": 20},

```

Comment:
I am not sure about replacing the old search space completely. Shall we keep the old search space in an alternative estimator? We'll need to compare the two search spaces later.

Reply:
We should do that in the experiment branch and finalize with the one that wins. Here for the stable version it's better to fix to one space so users are not confused.

Reply:
Have you confirmed the new search space wins? I thought it's not settled yet.

Reply:
It is better than the discrete space on mrpc.